.github/workflows/update-project-field.yml

```diff
@@ -0,0 +1,71 @@
+name: Update Project Field to Match Label
+
+# Triggers the workflow when a label is added to an issue.
+on:
+  issues:
+    types:
+      - labeled
+
+# Details needed to update a project field.
+env:
+  ORGANIZATION: "Automattic"
+  PROJECT_NUMBER: 391
+  PRIORITY_PREFIX: "[Pri]"
+  PRIORITY_FIELD: "Priority"
+
+jobs:
+
+  # Updates project fields to match label changes.
+  update_project_field: 
+    runs-on: ubuntu-latest
+    steps:
+
+      # Checks if the issue is already in the project. If not, the next steps will be skipped.
+      - name: Check if issue is in project
```

Comment:
**Suggestion (non-blocking):** If you want, you could skip this step too if the label doesn't have the right prefix! Not required though of course. 😄